src/main/resources/com/google/api/codegen/nodejs/test.snip

```diff
@@ -109,3 +116,42 @@
     });
   });
 @end
+
+@private streamingTestCase(test, moduleName)
+  describe('{@test.clientMethodName}', function() {
+    it('invokes {@test.clientMethodName} without error', function(done) {
+      var client = {@moduleName}.{@test.serviceConstructorName}();
+      // Mock request
+      {@initCodeLines(test.initCode)}
+
+      // Mock response
+      {@initCodeLines(test.mockResponse.initCode)}
+
+      // Mock Grpc layer
+      client._{@test.clientMethodName} = function() {
+        return new MockStream(request, expectedResponse);
+      };
+
+      @switch test.grpcStreamingType
+      @case "ServerStreaming"
+        client.{@test.clientMethodName}(request).on('data', function(response) {
+          assert.deepStrictEqual(response, expectedResponse);
+          done()
+        }).on('error', function(err) {
+          done(err);
+        });
```

Comment:
A change will need to be made to MockStream for the `'data'` event to be emitted for server streams. As it is right now, the event wont be called.